docs/live/README.md

```diff
@@ -13,13 +13,13 @@ Reconcile configuration files with the live state
 
 ### Synopsis
 
-Tools to safely apply and remove packages from clusters.
+Tool to safely apply and delete kubernetes package resources from live clusters.
 
-| Command   | Description                                       |
-|-----------|---------------------------------------------------|
-| [apply]   | apply a package to the cluster                    |
-| [preview] | preview the operations that apply will perform    |
-| [destroy] | remove the package from the cluster               |
+| Command   | Description                                               |
+|-----------|-----------------------------------------------------------|
+| [apply]   | apply a package to the live cluster                       |
+| [preview] | preview the operations that apply or destroy will perform |
```

Comment:
I think this should be 'preview the operations that will be executed by apply/destroy'

Reply:
Done